examples/p2p-data/script.js

```diff
@@ -0,0 +1,90 @@
+const Peer = window.Peer;
+
+(async function main() {
+  const localId = document.querySelector('[data-local-id]');
+  const localText = document.querySelector('[data-local-text]');
+  const connectTrigger = document.querySelector('[data-connect-trigger]');
+  const closeTrigger = document.querySelector('[data-close-trigger]');
+  const sendTrigger = document.querySelector('[data-send-trigger]');
+  const remoteId = document.querySelector('[data-remote-id]');
+  const messages = document.querySelector('[data-messages]');
```

Comment:
I want to know why you use data attributes to select unique elements instead of id?

Reply:
It certainly comes from my experience, but I can't remember its details now..

> https://twitter.com/leader22/status/1090877983634579456

And there is no significant problems to replace them by `id`, I'll fix it later.

Reply:
f6fdc36